applications/nrf_desktop/README.rst

```diff
@@ -377,41 +376,63 @@ Check :ref:`nrf_desktop_porting_guide` for details.
-For example, the ``release`` build type file name is :file:`prj_release.conf`.
-If a board has other configuration files, for example associated with partition layout or child image configuration, these follow the same pattern.
-
-.. include:: /config_and_build/modifying.rst
-   :start-after: build_types_overview_start
-   :end-before: build_types_overview_end
+The nRF Desktop application does not use a single :file:`prj.conf` file.
+Before you start testing the application, you can select one of the build types supported by the application, depending on your development kit and the building method.
+Not every board supports both mentioned build types.
+
+See :ref:`app_build_additions_build_types` and :ref:`modifying_build_types` for more information about this feature of the |NCS|.
+
+The application supports the following build types:
+
+.. list-table:: nRF Desktop build types
+   :widths: auto
+   :header-rows: 1
+
+   * - Build type
+     - File name
+     - Supported board
+     - Description
+   * - Debug
+     - :file:`prj.conf`
+     - All from `Requirements`_
+     - Debug version of the application; the same as the release build type, but with debug options enabled.
+   * - Release
+     - :file:`prj_release.conf`
+     - All from `Requirements`_
+     - Release version of the application with no debugging features.
+   * - Debug Fast Pair
+     - :file:`prj_fast_pair.conf`
+     - ``nrf52840dk_nrf52840``, ``nrf52840gmouse_nrf52840``
+     - Debug version of the application with `Fast Pair`_ support.
+   * - Release Fast Pair
+     - :file:`prj_release_fast_pair.conf`
+     - ``nrf52kbd_nrf52832``, ``nrf52840gmouse_nrf52840``
+     - Release version of the application with `Fast Pair`_ support.
+   * - Dongle
+     - :file:`prj_dongle.conf`
+     - ``nrf52840dk_nrf52840``
+     - Debug version of the application that lets you generate the application with the dongle role.
+   * - Keyboard
+     - :file:`prj_keyboard.conf`
+     - ``nrf52840dk_nrf52840``
+     - Debug version of the application that lets you generate the application with the keyboard role.
+   * - MCUboot QSPI
+     - :file:`prj_mcuboot_qspi.conf`
+     - ``nrf52840dk_nrf52840``
+     - Debug version of the application that uses MCUboot with the secondary slot in the external QSPI FLASH.
+   * - MCUboot SMP
+     - :file:`prj_mcuboot_smp.conf`
+     - ``nrf52840dk_nrf52840``
+     - | Debug version of the application that enables MCUmgr with DFU support and offers support for the MCUboot DFU procedure over SMP.
+       | See the :ref:`nrf_desktop_bootloader_background_dfu` section for more information.
+   * - WWCB
+     - :file:`prj_wwcb.conf`
+     - ``nrf52840dk_nrf52840``
+     - Debug version of the application with the support for the B0 bootloader enabled for `Works With ChromeBook (WWCB)`_.
 
 .. note::
-    `Selecting a build type`_ is optional.
-    The ``debug`` build type is used by default in nRF Desktop if no build type is explicitly selected.
```

Comment:
Maybe it's still worth mentioning? (`debug` build type uses `prj.conf` file)

Reply:
Right. Re-added in the build type section.